packages/vector_graphics/lib/src/listener.dart

```diff
@@ -746,7 +746,8 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
     listener = ImageStreamListener(
       (ImageInfo image, bool synchronousCall) {
         cacheCompleter.removeListener(listener);
-        _images[imageId] = image.image;
+        _images[imageId] = image.image.clone();
```

Comment:
I don't really understand what this is doing?

Reply:
What are your thoughts on this @jonahwilliams ?

Reply:
Making changes to the library to work around leak tracker is not reasonable.

Reply:
The way I understood it was that the leak tracker detected a memory leak in the library (`ImageInfo` is never being disposed) and what I did here was an attempt to fix this memory leak. 

Are you saying this is
1. Not a memory leak but a false positive? In that cases where is the `ImageInfo` usually disposed in production and would you know what is missing to the test to get it disposed too?
2. This is a memory leak but my fix is not the proper way to do it? In that case, would you have suggestions of what I could do instead?

Reply:
I don't know if this is a memory leak or a false positive (likely the latter), but even if this was memory leak I don't understand how to proposed change fixes it. it seems like what you're actually doing is working around leak tracker.

Reply:
For context, this is the failing test: 

https://github.com/flutter/packages/blob/d6705de7e4f15bfb89c8b9d2f5a47098d995eb27/packages/vector_graphics/test/vector_graphics_test.dart#L533-L592

At the end of this test, an `ImageInfo` remains undisposed. At first glance, I don't see anything wrong with the test itself, so this appears to be a memory leak.

Currently, `FlutterVectorGraphicsListener` only stores `image.image` and disposes of it in its `toPicture` method, leaving the outer `ImageInfo` unmanaged. I attempted to address this by disposing of the `ImageInfo` immediately.

Would it make sense to store the `ImageInfo` instead of just the `Image` to ensure everything gets properly disposed of in `FlutterVectorGraphicsListener.toPicture`?

Reply:
I pushed the change in [Store and dispose ImageInfo](https://github.com/flutter/packages/pull/8373/commits/86fc5367e5cd171ecc24b6c6cab0d5ab49480bad)

Reply:
That makes more sense